libraries/AP_Scripting/generator/src/main.c

```diff
@@ -31,6 +31,7 @@ char keyword_attr_literal[] = "'literal";
 char keyword_attr_null[]    = "'Null";
 char keyword_attr_reference[]  = "'Ref";
 char keyword_attr_array[]      = "'array";
+char keyword_attr_no_range_check[] = "'no_check";
```

Comment:
maybe rename to NoRangeCheck or NO_RNG_CHECK or something?

Reply:
or we could make it a positive comment like, "skip_check"

Reply:
done, changed to `skip_check`